modules/oc-mirror-configuring-credentials-v2.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-configuring-credentials-v2_{context}"]
+== Configuring credentials that allow images to be mirrored
+
+Create a container image registry credentials file that allows mirroring images from Red Hat to your mirror registry.
+
+[IMPORTANT]
+====
+Do not use this image registry credentials file as the pull secret when you install a cluster. If you provide this file when you install a cluster, all of the machines in the cluster will have write access to your mirror registry.
+====
+
+Ensure that you have write access to the container image registry on the mirror registry and adds the credentials to a registry pull secret.
```

Comment:
@subhtk container image registry on the mirror registry is confusing. can you  help check that ?